exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -211,9 +217,76 @@ public void testNoSupportedError() throws Exception {
         .go();
   }
 
+  @Test // DRILL-4530
+  public void testDrill4530_1() throws Exception {
+    // create metadata cache
+    test(String.format("refresh table metadata dfs_test.`%s/%s`", getDfsTestTmpSchemaLocation(), tableName2));
+    checkForMetadataFile(tableName2);
+
+    // run query and check correctness
+    String query1 = String.format("select dir0, dir1, o_custkey, o_orderdate from dfs_test.`%s/%s` " +
+            " where dir0=1995 and dir1='Q3'",
+        getDfsTestTmpSchemaLocation(), tableName2);
+    int expectedRowCount = 20;
+    int expectedNumFiles = 2;
+
+    int actualRowCount = testSql(query1);
+    assertEquals(expectedRowCount, actualRowCount);
+    String numFilesPattern = "numFiles=" + expectedNumFiles;
+    String usedMetaPattern = "usedMetadataFile=true";
+    String cacheFileRootPattern = String.format("%s/%s/1995/Q3", getDfsTestTmpSchemaLocation(), tableName2);
```

Comment:
The verification of cacheFileRootPattern probably need put "cacheFileRoot=" as the prefix. Otherwise,  the list of files in GroupScan will always find a match for cacheFileRoot, right?